src/lexer.rs

```diff
@@ -201,4 +201,13 @@ mod tests {
 		assert_eq!(&Lexer::new("(1.0,)").collect::<Vec<_>>(), &[Ok(Token::BracketOpen),
 			Ok(Token::Numeric(1.0)), Ok(Token::ListSeparator), Ok(Token::BracketClose)]);
 	}
+
+	#[test]
+	fn lexer_peek() {
+		let mut lexer = Lexer::new(r#"with "string""#);
+		assert_eq!(lexer.peek(), Ok(Some(&Token::Identifier(String::from("with")))));
+		assert_eq!(lexer.identifier().unwrap(), String::from("with"));
```

Comment:
Avoid unwrapping in an assertion and instead match against `Ok(...)`.

Reply:
Done